sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs

```diff
@@ -260,7 +260,7 @@ public virtual async Task CloseAsync(CancellationToken cancellationToken = defau
 
             if (maxWaitTime.HasValue)
             {
-                Argument.AssertPositive(maxWaitTime.Value, nameof(maxWaitTime));
+                Argument.AssertNotNegative(maxWaitTime.Value, nameof(maxWaitTime));
```

Comment:
I wonder if we should only allow TimeSpan.Zero when prefetch is enabled, since otherwise it will get replaced with the default value from the AMQP lib, right?

Reply:
Yes, right now it will be replaced with default value.
Sure, I will update logic here to check if prefetch mode is enabled